src/test/Test.FAKECore/ReleaseNotesSpecs.cs

```diff
@@ -132,7 +132,7 @@ public class when_parsing_many_pre_release_versions
 #### 2.0.0-alpha001 - December 15 2013
 * A";
         static readonly ReleaseNotesHelper.ReleaseNotes expected =
-            ReleaseNotesHelper.ReleaseNotes.New("2.0.0", "2.0.0-rc007", null, new[] { "A" }.ToFSharpList());
+            ReleaseNotesHelper.ReleaseNotes.New("2.0.0", "2.0.0-rc2", null, new[] { "A" }.ToFSharpList());
```

Comment:
Current logic is wrong, though.  Part of the reason I chafed working with nuget was because they implemented almost-sorta-but-not-really-semver, so none of the existing libraries worked.  Do we want to do the same?  Besides, if people depend on that sorting, they can either 

1. use a prerelease of the form `rc.{increment}`, where increment is a number and thus compared numerically, or
2. use a prerelease of the form `rc{padding}{number}`, and be consistent, so `rc007` compared to `rc002`,